inputfiles/removedTypes.jsonc

```diff
@@ -166,6 +166,14 @@
             "Text": {
                 "implements": ["GeometryUtils"]
             },
+            // This is hard to get in sync with @types/node
+            "AbortSignal": {
+                "methods": {
+                    "method": {
+                        "abort": null
```

Comment:
I think this one should be preferred over node, as Node is just following the web standard? What happened there?